.gitignore

```diff
@@ -103,6 +103,7 @@ celerybeat.pid
 
 # Environments
 .env
+!frontend/.env
```

Comment:
I used separate development env file originally for the token so that it is not included to the production build as it should not go there at all. So the intention was to have one env file for every build and one local env file for token in only development builds. The current simplification does not do this anymore. I think I should move the current .env.local back to .env.development.local file. I see there is .env and .env.template in the backend, but in that pattern the token would easily end up to the production build.

Reply:
It's changed to use only .env and .env.template files